go/vt/vtctl/workflow/server.go

```diff
@@ -1249,286 +1246,286 @@ func (s *Server) ReshardCreate(ctx context.Context, req *vtctldatapb.ReshardCrea
-}
-
-// VDiffShow is part of the vtctlservicepb.VtctldServer interface.
-func (s *Server) VDiffShow(ctx context.Context, req *vtctldatapb.VDiffShowRequest) (*vtctldatapb.VDiffShowResponse, error) {
-	span, ctx := trace.NewSpan(ctx, "workflow.Server.VDiffShow")
-	defer span.Finish()
-
-	span.Annotate("keyspace", req.TargetKeyspace)
-	span.Annotate("workflow", req.Workflow)
-	span.Annotate("argument", req.Arg)
-
-	tabletreq := &tabletmanagerdatapb.VDiffRequest{
-		Keyspace:  req.TargetKeyspace,
-		Workflow:  req.Workflow,
-		Action:    string(vdiff.ShowAction),
-		ActionArg: req.Arg,
-	}
-
-	ts, err := s.buildTrafficSwitcher(ctx, req.TargetKeyspace, req.Workflow)
-	if err != nil {
-		return nil, err
-	}
-
-	output := &vdiffOutput{
-		responses: make(map[string]*tabletmanagerdatapb.VDiffResponse, len(ts.targets)),
-		err:       nil,
-	}
-	output.err = ts.ForAllTargets(func(target *MigrationTarget) error {
-		resp, err := s.tmc.VDiff(ctx, target.GetPrimary().Tablet, tabletreq)
-		output.mu.Lock()
-		defer output.mu.Unlock()
-		output.responses[target.GetShard().ShardName()] = resp
-		return err
-	})
-	if output.err != nil {
-		s.Logger().Errorf("Error executing vdiff show action: %v", output.err)
-		return nil, output.err
-	}
-	return &vtctldatapb.VDiffShowResponse{
-		TabletResponses: output.responses,
-	}, nil
-}
-
-// VDiffStop is part of the vtctlservicepb.VtctldServer interface.
-func (s *Server) VDiffStop(ctx context.Context, req *vtctldatapb.VDiffStopRequest) (*vtctldatapb.VDiffStopResponse, error) {
-	span, ctx := trace.NewSpan(ctx, "workflow.Server.VDiffStop")
-	defer span.Finish()
-
-	targetShards := req.GetTargetShards()
-
-	span.Annotate("keyspace", req.TargetKeyspace)
-	span.Annotate("workflow", req.Workflow)
-	span.Annotate("uuid", req.Uuid)
-	span.Annotate("target_shards", targetShards)
-
-	tabletreq := &tabletmanagerdatapb.VDiffRequest{
-		Keyspace:  req.TargetKeyspace,
-		Workflow:  req.Workflow,
-		Action:    string(vdiff.StopAction),
-		VdiffUuid: req.Uuid,
-	}
-
-	ts, err := s.buildTrafficSwitcher(ctx, req.TargetKeyspace, req.Workflow)
-	if err != nil {
-		return nil, err
-	}
-
-	if len(targetShards) > 0 {
-		if err := applyTargetShards(ts, targetShards); err != nil {
-			return nil, err
-		}
-	}
-
-	err = ts.ForAllTargets(func(target *MigrationTarget) error {
-		_, err := s.tmc.VDiff(ctx, target.GetPrimary().Tablet, tabletreq)
-		return err
-	})
-	if err != nil {
-		s.Logger().Errorf("Error executing vdiff stop action: %v", err)
-		return nil, err
-	}
-
-	return &vtctldatapb.VDiffStopResponse{}, nil
-}
+// // VDiffCreate is part of the vtctlservicepb.VtctldServer interface.
+// // It passes on the request to the target primary tablets that are
+// // participating in the given workflow and VDiff.
+// func (s *Server) VDiffCreate(ctx context.Context, req *vtctldatapb.VDiffCreateRequest) (*vtctldatapb.VDiffCreateResponse, error) {
+// 	span, ctx := trace.NewSpan(ctx, "workflow.Server.VDiffCreate")
```

Comment:
We want to remove all of this commented code that was removed, don't we?

Reply:
Oops sorry! I don't know how this was left in self-review :(